audio_service/android/src/main/java/com/ryanheise/audioservice/AudioServicePlugin.java

```diff
@@ -438,52 +471,52 @@ public void onMethodCall(MethodCall call, final Result result) {
-                    }
-                    flutterReady = true;
-                    Map<?, ?> args = (Map<?, ?>)call.arguments;
-                    Map<?, ?> configMap = (Map<?, ?>)args.get("config");
-                    AudioServiceConfig config = new AudioServiceConfig(context.getApplicationContext());
-                    config.androidNotificationClickStartsActivity = (Boolean)configMap.get("androidNotificationClickStartsActivity");
-                    config.androidNotificationOngoing = (Boolean)configMap.get("androidNotificationOngoing");
-                    config.androidResumeOnClick = (Boolean)configMap.get("androidResumeOnClick");
-                    config.androidNotificationChannelId = (String)configMap.get("androidNotificationChannelId");
-                    config.androidNotificationChannelName = (String)configMap.get("androidNotificationChannelName");
-                    config.androidNotificationChannelDescription = (String)configMap.get("androidNotificationChannelDescription");
-                    config.notificationColor = configMap.get("notificationColor") == null ? -1 : getInt(configMap.get("notificationColor"));
-                    config.androidNotificationIcon = (String)configMap.get("androidNotificationIcon");
-                    config.androidShowNotificationBadge = (Boolean)configMap.get("androidShowNotificationBadge");
-                    config.androidStopForegroundOnPause = (Boolean)configMap.get("androidStopForegroundOnPause");
-                    config.artDownscaleWidth = configMap.get("artDownscaleWidth") != null ? (Integer)configMap.get("artDownscaleWidth") : -1;
-                    config.artDownscaleHeight = configMap.get("artDownscaleHeight") != null ? (Integer)configMap.get("artDownscaleHeight") : -1;
-                    config.setBrowsableRootExtras((Map<?,?>)configMap.get("androidBrowsableRootExtras"));
-                    if (activity != null) {
-                        config.activityClassName = activity.getClass().getName();
-                    }
-                    config.save();
-                    if (AudioService.instance != null) {
-                        AudioService.instance.configure(config);
-                    }
-                    mainClientInterface = ClientInterface.this;
-                    if (audioHandlerInterface == null) {
-                        audioHandlerInterface = new AudioHandlerInterface(messenger);
-                        AudioService.init(audioHandlerInterface);
-                    } else {
-                        if (audioHandlerInterface.messenger != messenger) {
-                            // We've detected this is the real engine hosting the AudioHandler,
-                            // so update AudioHandlerInterface to connect to it.
-                            audioHandlerInterface.switchToMessenger(messenger);
+                    case "configure":
```

Comment:
This change to formatting makes the diff very difficult to read and figure out what is "diff"erent. Please undo the formatting changes and only modify the lines that are relevant to the PR.